README.md

```diff
@@ -21,6 +21,16 @@ See [the SQL developer tutorial] to develop an app with C#, Java, Node.js, PHP,
 
 <img src="https://github.com/Microsoft/vscode-mssql/raw/main/images/mssql-demo.gif" alt="demo" style="width:480px;"/>
 
+## Version 1.13.0
+* Release date: February 24, 2022
+* Release status: GA
+
+## What's new in 1.13.0
+* Adds object explorer table integration for SQL bindings with Azure Functions
```

Comment:
`Adds context menu item for object explorer table nodes to create an Azure Function with SQL binding for that table`

@lucyzhang929 @chlafreniere @dzsquared This look better to you?

Reply:
I copied this from the text Drew had that Chris pointed me towards https://github.com/microsoft/vscode-mssql/issues/17263